locale/cs_CZ/foreman.po

```diff
@@ -11819,3 +11819,6 @@ msgstr ""
 
 msgid "you can't change administrator flag"
 msgstr ""
+
+msgid "This user is disabled and won't be able to perform any actions."
```

Comment:
Thanks for translating the strings, however we use external translation system for it, so these files are automatically generated and should not be modified directly by developers. Please take a look at https://projects.theforeman.org/projects/foreman/wiki/Translating for more details about how the translation works in Foreman project.